client/blocks/eligibility-warnings/hold-list.jsx

```diff
@@ -71,7 +71,13 @@ function getHoldMessages( siteSlug, translate ) {
 			description: translate(
 				'Please check your email to confirm your address.'
 			),
-		}
+		},
+		EXCESSIVE_DISK_SPACE: {
+			title: translate( 'Excessive diskspace' ),
+			description: translate(
+				'This site cannot be transferred due to diskspace limitations.'
```

Comment:
I think that's also confusing, if your site already has 350GB and is working fine before trying to install the plugin :/